src/main/java/fi/jubic/quanta/controller/ExternalClientController.java

```diff
@@ -52,6 +88,10 @@ public ExternalClient deleteExternalClient(Long externalClientId) {
         );
     }
 
+    public List<ExternalClient> getAllOfExternalClients() {

```

Comment:
Please use `getExternalClients` or something similar. This is just verbose.

Reply:
Changed to `getExternalClients`